misc/acrn-config/launch_config/com.py

```diff
@@ -276,32 +275,34 @@ def uos_launch(names, args, vmid, config):
     uos_type = names['uos_types'][vmid]
     launch_uos = launch_cfg_lib.undline_name(uos_type).lower()
 
+    # runc_container is clearlinux uos type, and apl-up2 dose not support for now
     if uos_type in ("CLEARLINUX", "ANDROID", "ALIOS") and not is_nuc_clr(names, vmid):
```

Comment:
does apl-up2 equals is_nuc_clr?

Reply:
Also a typo in the commit message "luanch" -> "launch"

Reply:
apl-up2 does not equals is_nuc_clr, is_nuc_clr is the nuc board with clearlinux uos.
The typo will be corrected.